### DISCUSSION TITLE
using snakemake as an option

### DISCUSSION THREAD

**Discussion by racfak** [drivendataorg/cookiecutter-data-science#396] [Requested tools]
Hi, 
is it possible to use snakemake as an option,
I found little tutorial on makefile in youtube and most are for C related.

**Comment by pjbull** [comment]
Latest version of snakemake is much improved since the last time I looked at it. Looks like a solid command runner.

Only open question is what's the best way to bootstrap snakemake. Maybe only thing we can do is add a system-level prerequisite in the default README.md instructing a `pipx` installation of snakemake.
